file-upload-types.php

```diff
@@ -61,17 +63,15 @@ function file_upload_types_deactivate_msg() {
 define( 'FILE_UPLOAD_TYPES_VERSION', '1.0.0' );
 
 /**
- * Return the main instance of File_Upload_Types.
+ * Return the main instance of Core class.
  *
  * @since 1.0.0
  *
- * @return File_Upload_Types
+ * @return Core
  */
 function file_upload_types() {
 
-	require_once dirname( __FILE__ ) . '/includes/class-file-upload-types.php';
-
-	return File_Upload_Types::get_instance();
+	return \File_Upload_Types\Core::get_instance();
```

Comment:
Instead of 
```php
return \File_Upload_Types\Core::get_instance();
```
it's better to have this:
```
$instance = \File_Upload_Types\Core::get_instance();
$instance->init();

return $instance;
```

This will allow removing all hooks from `Core::__construct()` method, which is good for testability and future maintenance.